patches/node/.patches

```diff
@@ -32,3 +32,6 @@ remove_makeexternal_case_for_uncached_internal_strings.patch
 fix_remove_outdated_--experimental-wasm-bigint_flag.patch
 fix_parallel_test-crypto-ecdh-convert-key_to_use_compatible_group.patch
 src_inline_asynccleanuphookhandle_in_headers.patch
+fix_handle_new_tostring_behavior_in_v8_serdes_test.patch
+fix_the_--harmony-weak-refs_has_been_removed_remove_from_specs.patch
```

Comment:
In short, these two lines have to be removed.

Reply:
Thanks @codebytere . You rock!